x-pack/plugin/eql/src/test/java/org/elasticsearch/xpack/eql/execution/sequence/CircuitBreakerTests.java

```diff
@@ -306,6 +223,107 @@ private void assertMemoryCleared(int sequenceFiltersCount, BiFunction<CircuitBre
         }
     }
 
+    public void testParentCircuitBreakerOnClean() {
```

Comment:
The test name imo is not descriptive enough. I'd suggest something like `testEqlCBCleanedUp_on_ParentCBBreak`.

Also, please add a comment to the method:
`// test covering fix for https://github.com/elastic/elasticsearch/issues/88300`